manager/src/main/java/com/redhat/service/smartevents/manager/api/user/ProcessorsAPI.java

```diff
@@ -112,6 +122,12 @@ public Response listProcessors(@NotEmpty @PathParam("bridgeId") String bridgeId,
     public Response addProcessorToBridge(@NotEmpty @PathParam("bridgeId") String bridgeId, @Valid ProcessorRequest processorRequest) {
         String customerId = identityResolver.resolve(jwt);
         String owner = identityResolver.resolveOwner(jwt);
+        String orgId = identityResolver.resolveOrganisationId(jwt);
+
+        // validate service limit
```

Comment:
@vaibhavjainwiz For consistency, I think we should move this into the `ProcessorService`. That leaves the `BridgeAPI` and `ProcessorsAPI` as simple controllers that expose our functionality via a HTTP endpoint. The actualy logic is all contained centrally within the `Service` implementations.

Reply:
done